### PR TITLE
Add CLI support for new Green LED APIs

### PR DESCRIPTION
**Pull request by mdegat01**
Add CLI support for the new APIs to set LEDs on Green added in https://github.com/home-assistant/supervisor/pull/4556

### CODE REVIEW

**Review by pvizeli — APPROVED** [review]

File: cmd/os_boards_green_options.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+			fmt.Println(err)
+			ExitWithError = true
+		} else {
+			ExitWithError = !helper.ShowJSONResponse(resp)
+		}
+	},
+}
+
+func init() {
+	osBoardsGreenOptionsCmd.Flags().Bool("activity-led", true, "Enable/disable the activity LED")
```
Comment (agners) [inline cmd/os_boards_green_options.go:53]:
```suggestion
	osBoardsGreenOptionsCmd.Flags().Bool("activity-led", true, "Enable/disable the green activity LED")
```

File: cmd/os_boards_green_options.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	osBoardsGreenOptionsCmd.Flags().Bool("power-led", true, "Enable/disable the power LED")
```
Comment (agners) [inline cmd/os_boards_green_options.go:54]:
```suggestion
	osBoardsGreenOptionsCmd.Flags().Bool("power-led", true, "Enable/disable the white power LED")
```

File: cmd/os_boards_green_options.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	osBoardsGreenOptionsCmd.Flags().Bool("user-led", true, "Enable/disable the user LED")
```
Comment (agners) [inline cmd/os_boards_green_options.go:55]: We aligned on calling the LED by their function. For Yellow we defined this to be "system health" LED (see also https://green.home-assistant.io/documentation/interfaces/).

```suggestion
	osBoardsGreenOptionsCmd.Flags().Bool("system-health-led", true, "Enable/disable the yellow system health LED")
```

File: cmd/os_boards_green_options.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	osBoardsGreenOptionsCmd.Flags().Lookup("activity-led").NoOptDefVal = "true"
+	osBoardsGreenOptionsCmd.Flags().Lookup("power-led").NoOptDefVal = "true"
+	osBoardsGreenOptionsCmd.Flags().Lookup("user-led").NoOptDefVal = "true"
```
Comment (agners) [inline cmd/os_boards_green_options.go:58]:
```suggestion
	osBoardsGreenOptionsCmd.Flags().Lookup("system-health-led").NoOptDefVal = "true"
```

File: cmd/os_boards_green_options.go [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+		section := "os"
+		command := "boards/green"
+
+		options := make(map[string]interface{})
+
+		for _, value := range []string{
+			"activity-led",
+			"power-led",
+			"user-led",
```
Comment (agners) [inline cmd/os_boards_green_options.go:34]:
```suggestion
			"system-health-led",
```

**Review by agners — CHANGES_REQUESTED** [review]

**Review by agners — APPROVED** [review]

**Pull request merged**